scarlet/source.py

```diff
@@ -597,3 +707,80 @@ def make_initial(self, img, bg_rms, thresh=1., symmetric=True, monotonic=True, c
             # keep the peak sed
             logger.INFO("Using peak SED for source at {0}/{1}".format(self.center_int[0], self.center_int[1]))
         return sed.reshape((1,B)), morph.reshape((1, morph.shape[0], morph.shape[1]))
+
+
+class MultiComponentSource(ExtendedSource):
```

Comment:
I foresee the need in the future to make spiral galaxies with different stellar populations that *don't* share a common center multi-component sources, this is just a special case where the positions are aligned. We should talk a bit about what's the best way to handle the more general case, when sources aren't aligned, before we put too much work in this class.